src/actions/smartWalletActions.js

```diff
@@ -770,12 +771,7 @@ export const onSmartWalletSdkEventAction = (event: Object) => {
       const deployedAccountState = get(sdkConstants, 'AccountStates.Deployed', '');
       if (newAccountState === deployedAccountState && accountState !== deployedAccountState) {
         dispatch(setSmartWalletUpgradeStatusAction(SMART_WALLET_UPGRADE_STATUSES.DEPLOYMENT_COMPLETE));
-        Toast.show({
-          message: 'Your Smart Wallet has been deployed',
-          type: 'success',
-          title: 'Success',
-          autoClose: true,
-        });
```

Comment:
Deleted this Toast because it doesn't make much sense now that we have a whole screen informing about the same.